keps/prod-readiness/sig-node/2823.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 2823
+alpha:
+  approver: "@dchen1107"
```

Comment:
Node approver should be provided in the kep.yaml (like you already have). Here one of the approvers from the list of Production readiness approvers should be provided: https://github.com/kubernetes/enhancements/blob/0ec64c8aa3f1071ba5326cfeaadc702448d3a488/OWNERS_ALIASES#L191-L195